bobtemplates/django/+package.fullname+/setup.py.bob

```diff
@@ -0,0 +1,61 @@
+from setuptools import find_packages
+from setuptools import setup
+
+
+extras = {
+    'api': [
+        'djangorestframework',
+    ],
+    'core': [
+        'django',
+        'django-classy-settings',
+        'factory_boy',
+        'freezegun',
+        'psycopg2',
+        'python-dotenv',
+        'raven',
+    ],
+    'dev': [
+        'django-rest-swagger',
+    ],
+    'tests': [
+        'pytest',
+        'pytest-cov',
+        'pytest-django',
+        'pytest-dotenv',
+        'pytest-factoryboy',
+        'tox',
+    ],
+    'web': [
+    ],
+}
+
+VERSION = '1.0.0.dev0'
+GITHUB_URL = 'https://github.com/4teamwork/{{{package.fullname}}}'
+
+setup(
+    name='{{{package.fullname}}}',
+    version=VERSION,
+    license='',
+    url=GITHUB_URL,
+    download_url='{}/archive/v{}.tar.gz'.format(
```

Comment:
What's the reason you are doing that?
It could have negative effects when publishing to pypi: it probably implies that the tarballs are not hosted on pypi but on github.
I don't see the benefit and would remove it (unless you have a good reason 😉 )

Reply:
I see a use case, where the package is not on pypi and is installed from github.
But that should barely happen. I'm removing the download URL

Reply:
If the package is not on pypi, the download url is not relevant 😉  I'd remove it for simplicity - it can be added to a project if it is really relevant (although I have never released such a package).